### ISSUE TITLE
Save a mock twilio response to the database

### ISSUE THREAD

**Issue by thebritican** [thebritican/eviction-tracker#5]
## Context

Before we verify any phone numbers, we need to make sure we can save the verification data so we never lose what we've paid for. We are building a database of information: detainer warrants, attorneys, defendants, etc. There will also be a table for `phone_number_verifications`. We will then always be able to look up our data from Twilio in that database.

## Description

This should serve as a test that our code will do the right thing when eventually validating a phone number.

We should make sure that the [JSON response documented by Twilio](https://www.twilio.com/docs/lookup/tutorials/carrier-and-caller-name?code-sample=code-lookup-with-e164-formatted-number&code-language=Python&code-sdk-version=6.x) is able to be saved to the database.

### Example JSON response
```
{
  "caller_name": {
    "caller_name": "Delicious Cheese Cake",
    "caller_type": "CONSUMER",
    "error_code": null
  },
  "carrier": null,
  "country_code": "US",
  "national_format": "(510) 867-5310",
  "phone_number": "+15108675310",
  "add_ons": null,
  "url": "https://lookups.twilio.com/v1/PhoneNumbers/+15108675310"
}
```

## Modeling

We need to build a database table to properly model this data before inserting it.

I suggest:

```python
class PhoneNumberVerification(Base):
    __tablename__ = 'phone_number_verifications'
    id = Column(Integer, foreign_key=True)
    caller_name = Column(String)
    caller_type = Column(Integer) # smaller column than String
    error_code = Column(Integer)
    carrier = Column(String)
    country_code = Column(String)
    national_format = Column(String)
    phone_number = Column(String)
```

This is actively under construction. Follow progress in #6.

## Inserting

Please follow SQLAlchemy docs for how to insert [this kind of data to the database](https://docs.sqlalchemy.org/en/13/orm/tutorial.html#adding-and-updating-objects)

## Testing

We don't yet have a test suite, i'll comment on this issue when we do. When that is set up, we can write a test to close out this issue:

1. Create mock JSON data
2. Test that the JSON can be inserted into the `phone_number_verifications` table

**Comment by thebritican** [comment]
@donald-mate What we'll need for an initial test is:

1. A method that takes JSON and turns it into `PhoneNumberVerification` entry

```python
class PhoneNumberVerification(db.Model):
    ...
    def from_twilio_response(json):
      return PhoneNumberVerification(caller_name=json.caller.name) # something like this with the rest of the fields handled

```

2. a fake piece of JSON data like in the description
3. a test that asserts that `from_twilio_response` creates a valid `PhoneNumberVerification`

```
input = some_json
output = PhoneNumberVerification.from_twilio_response(input)

# then we test
assert_equal(input.caller.name, output.caller_name)
assert_equal(input.caller.type output.caller_type)
...
```

**Comment by thebritican** [comment]
Let's start with the python standard testing framework for #11 

https://docs.python.org/3/library/unittest.html

But if we get that in, i'd love to try out [Hypothesis](https://hypothesis.readthedocs.io/en/latest/quickstart.html#writing-tests). Basically, Hypothesis let's you generate test input data. So, instead of having to write out individual variations of JSON you want to test, you have the Hypothesis library randomly create data for your test.

**Comment by thebritican** [comment]
solved with #16
